config/locales/en.yml

```diff
@@ -385,8 +385,23 @@ en:
       text: "Guidance on trapped capital"
       link: "https://legalaidlearning.justice.gov.uk/course/view.php?id=186"
     disregarded_benefits:
-      text: Disregarded benefits
-      hint: The following benefits are disregarded from the financial eligibility calculation
+      text: Government Cost of Living Payments and disregarded benefits
+      cost_of_living:
+        text: Government Cost of Living Payments
+        list:
+          - Government Cost of Living Payments are disregarded from the financial eligibility calculation, for example
```

Comment:
should this line end with a colon ":", its not in the ticket but it does precede a list, which is where we would usually use one.

Reply:
I'll double-check with Chris. You're right, we use a colon prior to a list in several places elsewhere.

Reply:
@willc-work Chris has confirmed that in this case he does _not_ want a colon.